Library/Formula/mpd.rb

```diff
@@ -116,4 +119,18 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork do
+      exec "#{bin}/mpd --stdout --no-daemon --no-config"
+    end
+    sleep 2
+
+    begin
+      assert_match /OK MPD/, shell_output("curl localhost:6600")
+    ensure
+      Process.kill("SIGINT", pid)
+      Process.wait(pid)
+    end
+  end
```

Comment:
Seemingly, It can be either and works. Homebrew's internal use tends to prefer `begin` inside the ` fork do`, formulae lean towards preferring the `begin` after the `fork do` :neutral_face:.